packages/ses/scripts/hermesTransforms.js

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { parse } from '@babel/parser';
+import babelGenerate from '@agoric/babel-generator';
+import babelTraverse from '@babel/traverse';
+import * as t from '@babel/types';
+
+// @ts-expect-error TS2339 Property 'default' does not exist
+const traverse = babelTraverse.default || babelTraverse;
+const generate = babelGenerate.default || babelGenerate;
+
+const decoder = new TextDecoder('utf-8');
+// @ts-expect-error TS2554 Expected 0 arguments, but got 1
+const encoder = new TextEncoder('utf-8');
+
+const asyncArrowEliminator = {
+  ArrowFunctionExpression(path) {
+    if (path.node.async) {
```

Comment:
```suggestion
    if (path.getData('async')) {
```

(untested)

Using the functions provided by the `NodePath` object instead of reaching into its `node` prop reduces coupling, FWIW.  Could use this wherever you are accessing `nodePath.node.<prop>`.

Reply:
`path.getData('async')` didn't work, but [spotted](https://github.com/user-attachments/assets/c11e94f2-927c-4662-8ae7-2a2466fb0b5e) it in the types :suspect:

`(path.get('async').node` works too, but slightly less efficient/simple/direct